src/promptflow/setup.py

```diff
@@ -32,7 +32,7 @@
     "pandas>=1.5.3,<3.0.0",  # load data requirements
     "python-dotenv>=1.0.0,<2.0.0",  # control plane sdk requirements, to load .env file
     "keyring>=24.2.0,<25.0.0",  # control plane sdk requirements, to access system keyring service
-    "pydash>=6.0.0,<7.0.0",  # control plane sdk requirements, to support parameter overrides in schema.
+    "pydash>=7.0.0,<8.0.0",  # control plane sdk requirements, to support parameter overrides in schema.
```

Comment:
MARK we don't need to increase the lower bound

Reply:
understood, you can regard this as a test commit, I will make this `>=6.0.0<8.0.0` after the verification

Reply:
updated